website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -527,7 +530,7 @@ The graph of this pipeline looks like the following:
 *Figure 1: A linear pipeline with three sequential transforms.*
 
 However, note that a transform *does not consume or otherwise alter* the input
-collection--remember that a `PCollection` is immutable by definition. This means
+collection - remember that a `PCollection` is immutable by definition. This means
```

Comment:
Em dash with no spaces between words is the correct usage here: —

Reply:
done